src/client/components/Utils/USDDisplay.js

```diff
@@ -3,13 +3,19 @@ import PropTypes from 'prop-types';
 import { FormattedNumber } from 'react-intl';
 
 const USDDisplay = ({ value }) => {
-  const negative = value.toFixed(2) < 0;
+  const negative = value < 0;
   const absValue = Math.abs(value);
+  // 0.02 is dust payout threshold (STEEM_MIN_PAYOUT_SBD)
+  const precision = absValue < 0.02 && value > 0 ? 3 : 2;
   return (
     <span>
       {negative && '-'}
```

Comment:
I think we could get rid of this and use `FormattedNumber` to display a negative sign if needed.

Reply:
Right, I forgot about the `$` sign.

It might not be important to display `-$0.013` instead of `-$0.01`, but I'd rather see `-$0.009` instead of `-$0.00` (we display `$0.009` for positive values).

Reply:
Okay I'll fix that. In another sense of consistency, you're right. I also actually implemented that way in the beginning :)

Reply:
![](https://user-images.githubusercontent.com/38183982/52051714-3943d600-254c-11e9-94f6-1f5a7be59887.png)

Done!

More test coded are also added for handling zero correctly. Thanks!

Also huge thanks for busy bot :) I also made a post about it: https://busy.org/@blockchainstudio/en-kr-busy-voting-independence-day-now-it-s-completely-independent-of-steemdb-busy-steemdb